packages/library-sync/src/commonMain/kotlin/io/realm/mongodb/internal/Provider.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.mongodb.internal
+
+/**
+ * This enum contains the list of identity providers supported by MongoDB Realm.
+ * All of these except [.EMAIL_PASSWORD] must be enabled manually on MongoDB Realm to
```

Comment:
Previously you also had to enable this one. It wasn't on as a default. Has this changed?

Reply:
I removed any mentioning of defaults but just highlighted that the providers need to be enabled in the app with a reference to docs on that.